pkg/certificate/certificate.go

```diff
@@ -46,11 +46,12 @@ func NewKeyPair(tlsGenerate bool, keyFile, certFile string) *Keypair {
 	}
 }
 
-func (k *Keypair) generate() error {
-	org := ""
+func CreateRawKeyPair() (bytes.Buffer, bytes.Buffer, error) {
```

Comment:
Yeah I agree, this totally needs comments. I know you called this minor further down the page but I actually disagree, especially since I didn't use named return values, it's pretty unclear.